pkg/reconciler/testing/v1/subscription.go

```diff
@@ -219,6 +225,12 @@ func WithSubscriptionPhysicalSubscriptionReply(uri *apis.URL) SubscriptionOption
 	}
 }
 
+func WithSubscriptionPhysicalSubscriptionReplyCACerts(caCerts string) SubscriptionOption {
+	return func(s *v1.Subscription) {
+		s.Status.PhysicalSubscription.ReplyCACerts = &caCerts
+	}
+}
```

Comment:
I wouldn't introduce those functions but change `WithSubscriptionDeadLetterSinkURI` to `WithSubscriptionDeadLetterSink` and take `duckv1.Addressable` in input